api.planx.uk/modules/gis/service/digitalLand.ts

```diff
@@ -186,7 +187,20 @@ async function go(
   }
 
   // --- LISTED BUILDINGS ---
-  // TODO add granular variables to reflect grade (eg `listed.grade1`), not reflected in content yet though
+  if (formattedResult["listed"] && formattedResult["listed"].value) {
+    ["listd.grade.I", "listed.grade.II", "listed.grade.II*"].forEach(
+      (grade) =>
+        (formattedResult[grade] = {
+          fn: grade,
+          value: Boolean(
+            formattedResult["listed"].data?.filter(
+              (entity) =>
+                entity["listed-building-grade"] === grade.split(".").pop(),
```

Comment:
My preferred granular passport variable names here were `listed.gradeI` rather than `listed.grade.I` - but then we can't check for a direct match against how Planning Data publishes "grades" - and "endsWith" doesn't work with `I` & `II`.

We hit a similar thing with introducing specific flood zone variables above - so at least the many segments is consistent.

Will update the ODP Schema PR to match !